orttraining/orttraining/python/ort_trainer.py

```diff
@@ -845,6 +879,63 @@ def _prepare_input_and_fetches(self, input_desc_with_, internal_learning_rate, i
 
         return input, fetches
 
+    def _compare_inference_output_pytorch_onnx(self, input, input_descs, output_desc):
+        # handle case if onnx model is provided instead of torch
+        if not self.torch_model_:
+            return
+
+        # set up training session for ort models (for inference)
+        no_postprocess_run_options = ort.RunOptions()
+        no_postprocess_run_options.only_execute_path_to_fetches = True
```

Comment:
We are not specifying `fetches=[]` on `train_step`/`eval_step`, so we probably dont need to specify `no_postprocess_run_options.only_execute_path_to_fetches=True` here